opentera-webrtc-native-client/OpenteraWebrtcNativeClient/python/package/__init__.py.in

```diff
@@ -1,3 +1,23 @@
-# Init
 from opentera.webrtc.native_client.${PYTHON_LIB_NAME} import *
-from opentera.webrtc.native_client.${PYTHON_LIB_NAME} import __version__
+
+def _patch():
+
+    __all_imports = {
+        _k: _v
+        for _k, _v
+        in globals().items()
+        if isinstance(_v, type) and isinstance(_k, str)
+        and _v.__module__ == 'opentera.webrtc.native_client._opentera_webrtc_native_client'
+    }
+
+    __all = []
+
+    for _k, _v in __all_imports.items():
+        _v.__module__ = _v.__module__.replace(
+            '.${PYTHON_LIB_NAME}', '')
+        __all.append(_k)
+
+    return __all
+
+
+__all__ = _patch()
```

Comment:
Dynamically generate a `__all__`, which allows `sphinx` to document the content of `opentera.webrtc.native_client._opentera_webrtc_native_client` as if it was directly in `opentera.webrtc.native_client`.